src/lm_buddy/jobs/__init__.py

```diff
@@ -0,0 +1,30 @@
+from lm_buddy.integrations.wandb import ArtifactLoader, WandbArtifactLoader
+from lm_buddy.jobs.common import LMBuddyJobConfig
+from lm_buddy.jobs.finetuning import FinetuningJobConfig, run_finetuning
+from lm_buddy.jobs.lm_harness import LMHarnessJobConfig, run_lm_harness
+from lm_buddy.jobs.simple import SimpleJobConfig, run_simple
+
+
+def run_job(
+    config: LMBuddyJobConfig,
+    *,
```

Comment:
do we always know we're only getting config and artifact loader here? if that's the case, we can consider dropping the args for clarity

Reply:
> we can consider dropping the args for clarity

What do you mean? I don't follow this.

Reply:
if we don't expect anything other than `config` and `artifact_loader`, likely we don't need the `*`

Reply:
Ahh, you mean making the artifact loader an arg instead of a kwarg. Sure, I can do that.

It's a bit of stylistic preference from old repos I've worked on, where we generally used kwargs as much as possible to make the call sites more clear. In this case, given that the `config` is the main argument and the `artifact_loader` is more of a dependency of the method, it felt natural.

But I don't mind much either way, ill change it.